frontend/projects/ui/src/app/util/properties.util.ts

```diff
@@ -149,12 +149,12 @@ function parsePropertiesV1Permissive(
       } else {
         const error = result.error
         const message = Parser.validatorErrorAsString(error)
-        let dataPath = error.keys.map(x => JSON.parse(x)).join('/')
+        const dataPath = error.keys.map(removeQuotes).join('/')
         errorCallback(new Error(`/data/${idx}: ${message}`))
         if (dataPath) {
           applyOperation(cur, {
             op: 'replace',
-            path: dataPath,
+            path: `/${dataPath}`,
```

Comment:
I probably shouldn't make this change. Paths should start with /
Just noticed it was wrong when looking at this

Reply:
No this is fine as it it complied with the specification